README.md

```diff
@@ -61,6 +61,15 @@ Past maintainers:
 * Daniel Molkentin <daniel@molkentin.de>
 * Andreas Schneider <asn@cryptomilk.org>
 
+## Building the Documentation
+
+The documentation has been migrated from Sphinx-Doc to [Antora](https://docs.antora.org/), which is based on [the AsciiDoc format](https://github.com/owncloud/docs/blob/master/docs/getting-started.md).
```

Comment:
I would reference via link else you have to maintain two sides.
Like:
Please see the README.md in https://github.com/owncloud/docs for more information